### PR TITLE
Add prometheus and alertmanager tunnels.

### PR DESCRIPTION
**Pull request by SchoolGuy**
This aims to close the issue #147

### CODE REVIEW

**Comment by SchoolGuy** [comment]
I tested it locally and it works on my machine. Is there the need of any additional automatic testing?

File: seslib/__init__.py [hunk 1]
```diff
@@ -1273,6 +1274,16 @@ def start_port_forwarding(self, service=None, node=None, remote_port=None, local
                 remote_port = 443
                 local_port = 8443
                 service_url = 'https://{}:{}'.format(local_address, local_port)
+            elif service == 'prometheus':
+                node = 'admin'
```
Comment (p-se) [inline seslib/__init__.py]: Prometheus and Alertmanager can probably be installed on other nodes than the `admin`. Maybe we should use `_find_service_node` here (and below), too.

Comment (SchoolGuy) [reply seslib/__init__.py]: Okay I will adjust this. Thanks for the feedback!

Comment (SchoolGuy) [reply seslib/__init__.py]: Resolved with 610187d

**Comment by smithfarm** [comment]
@SchoolGuy Can you squash the commits into a single commit, please?

There are lots of tutorials out there explaining how to do it. Here's how I do it:

```
$ git rebase -i HEAD~3
(in the editor, leave the first commit as "pick" and change the second and third to "squash")
(save and exit)
(edit the commit message)
(save and exit)
$ git push -f
```

**Comment by SchoolGuy** [comment]
@smithfarm Yeah of course, although I am a more graphical guy which means I will use my fancy Paycharm Tool for that! :blush:

**Comment by SchoolGuy** [comment]
@smithfarm Done

**Comment by smithfarm** [comment]
@SchoolGuy Thanks! Now, can you also do the Paycharm Tool equivalent of `git commit --amend --signoff`? ;-)

**Comment by smithfarm** [comment]
Also, please add a `Fixes: https://github.com/SUSE/sesdev/issues/147` line to the commit message. It should be near the bottom of the commit message, immediately above the `Signed-off-by` line.

**Comment by SchoolGuy** [comment]
@smithfarm This should now be good I think...

**Comment by smithfarm** [comment]
@SchoolGuy In the "Signed-off-by" line, please change "SchoolGuy" to your real name.

Also please read https://github.com/SUSE/sesdev/blob/42031d3f1e05684f5114258ed7a2d7d95c559769/CONTRIBUTING.rst so you know what you're agreeing to ;-)

**Comment by SchoolGuy** [comment]
@smithfarm Okay changed it to my real name now. Hopefully I did not forget something now.

**Comment by smithfarm** [comment]
@SchoolGuy @jan--f I am merely applying the same basic requirement that the Ceph project applies to all contributions. Hope it's not too much trouble for you.

**Comment by SchoolGuy** [comment]
@smithfarm Nope I never had the requirement to do it so I wasn't expecting it. But fine by me.

**Comment by smithfarm** [comment]
@SchoolGuy It's good preparation for your future first upstream Ceph contribution :-)

**Comment by SchoolGuy** [comment]
Okay this now revealed a problem... The Alertmanager has no dedicated role and since the `_find_service_node` fails for it. Is it reasonable to introduce a new role or do we need to do some magic to fix this?

**Comment by smithfarm** [comment]
@SchoolGuy Can we change "alertmanager" to "grafana"?

**Comment by SchoolGuy** [comment]
Nope alertmanager has its own port, its own service and is not bundled with anything. This can be independtly set up on any node (or even outside a node) and just needs access to prometheus via http...

**Comment by smithfarm** [comment]
@SchoolGuy Sounds like we (you?) need to add an alertmanager role in sesdev and in ceph-salt as well, then.

**Comment by SchoolGuy** [comment]
Wonderful thanks for the direction.

**Pull request merged**